oauth2_http/java/com/google/auth/oauth2/S2A.java

```diff
@@ -58,7 +58,7 @@
  * <p>This is an experimental utility.
  */
 @ThreadSafe
-public final class S2A {
+public class S2A {
```

Comment:
On a second look, do you have concerns if we can make this name written out specifically (SecureSessionAgent.java) and (SecureSessionAgentConfig.java) as S2A may not be well known and specific to Google?

Reply:
That's fine with me, done in https://github.com/googleapis/google-auth-library-java/pull/1573/commits/e6b6e18582b7538a2ea6082c13885355016fbcd4